core/vtk/ttkAlgorithm/ttkAlgorithm.cpp

```diff
@@ -42,6 +42,10 @@ ttk::Triangulation *ttkAlgorithm::GetTriangulation(vtkDataSet *dataSet) {
                  ttk::debug::Priority::DETAIL);
 #if TTK_ENABLE_MPI
   if(ttk::isRunningWithMPI()) {
+    if(!hasMPISupport_) {
```

Comment:
ok, that's the rightspot. 
the constructor could have been a (more verbose) alternative.
like this, it is fine :)